common/meta/types/src/meta_errors_into.rs

```diff
@@ -39,24 +38,12 @@ impl From<MetaError> for ErrorCode {
     }
 }
 
-impl From<MetaStorageError> for MetaError {
-    fn from(e: MetaStorageError) -> Self {
-        MetaError::MetaStorageError(e)
-    }
-}
-
 impl From<serde_json::Error> for MetaError {
     fn from(error: serde_json::Error) -> Self {
         MetaError::MetaStorageError(error.into())
     }
 }
 
```

Comment:
If MetaStorageError is a superset of serde json Error, maybe MetaError itself does not need to include serde json Error.

If it is true, maybe we could fix it in next PR 🤔